src/utils/swb_cranker.rs

```diff
@@ -60,24 +72,115 @@ impl SwbCranker {
             .enable_all()
             .build()?;
 
+        let crossbar_client =
+            CrossbarClient::new(&config.swb_crossbar_url, config.swb_crossbar_verbose);
+
         let rpc_client = RpcClient::new(config.rpc_url.clone());
         let non_blocking_rpc_client = NonBlockingRpcClient::new(config.rpc_url.clone());
         let queue = tokio_rt.block_on(QueueAccountData::load(
             &non_blocking_rpc_client,
-            &Pubkey::from_str(SWB_PROGRAM_ID).unwrap(),
+            &config.swb_program_id,
         ))?;
         let swb_gateway =
             tokio_rt.block_on(queue.fetch_gateways(&non_blocking_rpc_client))?[0].clone();
 
         Ok(Self {
+            cluster_type: config.cluster_type,
             tokio_rt,
             rpc_client,
             non_blocking_rpc_client,
+            crossbar_client,
             swb_gateway,
             payer,
         })
     }
 
+    pub fn simulate_stale_prices(&self, cache: &Cache) -> Result<()> {
+        thread_info!("Simulating stale Swb prices...");
+
+        let stale_oracles = self.find_stale_oracles(cache)?;
```

Comment:
Again re: above comment, it's easier logically to just sim all the swb oracles. Simming just the stale ones opens this up to race conditions e.g. another goes stale as this is running, etc.